### DISCUSSION TITLE
Scroll to new view without tabs

### DISCUSSION THREAD

**Discussion by aryehsilver** [dotnet/maui#15765] [General]
I'm building a home app for an Android device. What is the best way to replicate the experience on a typical mobile, I.e. have a home screen that displays the time and search box etc. Then when scrolling it opens a list of available apps.

I have tried with a TabbedPage. The problem with this, I haven't worked out how to remove the tabs. It should only be navigable via scrolling not clicking the tab.

The CarouselView only supports adding an ItemsTemplate which doesn't make sense here as I need different content on the home screen verses the apps list.

What is my best way forwards?

**Comment by aryehsilver** [comment]
Can no one help?

**Comment by aryehsilver** [comment]
Is MAUI dead? Or are all the developers and maintainers dead!?!
Why is no one replying even if only to say I have no idea how to help you?!?

**Comment by Eilon** [comment]
Hi, I converted this issue to a discussion post because it's not about a specific issue or feature request.
